cmd2.py

```diff
@@ -880,33 +881,6 @@ def postparsing_postcmd(self, stop):
         """
         return stop
 
-    def parseline(self, line):
```

Comment:
@catherinedevlin 
This is an override of a a method defined in cmd.  It is actually used within the parent cmd module in the complete() method.

Believe it or not it wasn't around from before, I just added this override a couple weeks ago.  

cmd2 needed to override it because cmd2 has more command shortcuts than cmd.

Maybe we should add something to the docstring to better explain its purpose.